txscript/sigcache_test.go

```diff
@@ -187,3 +201,51 @@ func TestSigCacheAddMaxEntriesZeroOrNegative(t *testing.T) {
 			"been added", len(sigCache.validSigs))
 	}
 }
+
+// TestEvictEntries tests that evictEntries properly removes all SigCache
+// entries related to the given block.
+func TestEvictEntries(t *testing.T) {
+	sigCache := NewSigCache(8, shortTxHashKey)
```

Comment:
Probably should make this `numTxns+1` so it's correct if the test block is ever changed.